scipy/sparse/tests/test_base.py

```diff
@@ -1354,6 +1354,16 @@ def check(dtype):
         for dtype in self.math_dtypes:
             check(dtype)
 
+    # github issue #15210
+    def test_rmul_scalar_type_error(self):
+        def check(dtype):
```

Comment:
Not sure it's important to test for different dtypes here; important to test for all matrix container types.